keycastr/KCKeyboardTap.m

```diff
@@ -170,6 +173,19 @@ - (void)dealloc {
     [super dealloc];
 }
 
+- (void)openPrefsPane {
+    NSString *text = @"tell application \"System Preferences\"   \n\
+    --get a reference to the Security & Privacy preferences pane \n\
+    set securityPane to pane id \"com.apple.preference.security\"\n\
+    tell securityPane to reveal anchor \"Privacy_Accessibility\" \n\
+    --open the preferences window and make it frontmost          \n\
+    activate \n\
+    end tell \n";
```

Comment:
Good call.  AppleScript always gives me the heebie-jeebies, but enough people are confused about this that it's bound to be helpful.